build/mathquill.js

```diff
@@ -2834,16 +3203,28 @@ var MathBlock = P(MathElement, function(_, super_) {
     return node.seek(pageX, cursor);
   };
   _.chToCmd = function(ch) {
+    ch = ch || ''
+    // We want to use the available commands from the controller
+    var controller, node = this
+    do {
+      controller = node.controller 
+      node = node.parent
+    } while(!controller)
+    
+    var latexCmds = controller.latexCmds
+    var charCmds = controller.charCmds
     var cons;
+    
     // exclude f because it gets a dedicated command with more spacing
-    if (ch.match(/^[a-eg-zA-Z]$/))
-      return Letter(ch);
+    if (ch.match(/^[a-eg-zA-Z\+\-\!\=]$/))
+      cons = Letter(ch);
     else if (/^\d$/.test(ch))
-      return Digit(ch);
-    else if (cons = CharCmds[ch] || LatexCmds[ch])
-      return cons(ch);
+      cons = Digit(ch);
+    else if (cons = controller.searchForCommand(ch) )
```

Comment:
**opinion** Assignment in an `if` is super nasty (pre-existing). Would be better to place some curlys on this block and do it properly. Meaning:
```
else {
  var fn = controller.searchForCommand(ch);
  return fn ? fn(ch) : VanillaSymbol(ch);
}
```

Reply:
wrong file. I'm happy with the src file.